src/kompos/__init__.py

```diff
@@ -37,14 +37,16 @@ def __call__(self, result, pass_trough=True, cwd=None):
                 color='dark gray')
 
     def _execute(self, result, pass_trough=True, cwd=None):
+        exit_code = 0
```

Comment:
Is this used?

Reply:
Yes, it's used in the `Executor` but was not referenced, I've notice this error while interrupting some runners.